docs/rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,126 @@
+# RFC-NNNN: Human-readable title goes here
+
+Remember to add autogenerated HackMD badge here. See [README.md](README.md) for more details.
+
+**Authors** (in alphabetical order):
+
+- Author Name, [@Author_GitHub_Handle](https://github.com/Author_GitHub_Handle)
```

Comment:
```suggestion
- Author Name, [@Author_GitHub_Handle](https://github.com/Author_GitHub_Handle)
- Foo Bar, [@Foo_Bar](https://github.com/Foo_Bar)
```
This is just to clarify, it's alphabetical order based on first name instead of surname.